package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
@abstractj I'm not sure we should have a package-lock directly in the root directory (where's no node_modules). How did this even get here? :) However, it's a non-blocking issue.